presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -310,11 +310,23 @@ Java Keystore File Verification
 Verify the password for a keystore file and view its contents using
 :ref:`troubleshooting_keystore`.
 
+SSL debugging for the server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To add debugging log messages from the Presto server about TLS/SSL connection
+errors, add the following line to the ``logging.properties`` file in the
+server's ``etc`` directory:
+
+.. code-block:: text
+
+    io.prestosql.plugin.password=DEBUG
```

Comment:
The title is "SSL debugging for the server", but this is actually about debugging issues with the Presto server talking to the LDAP server. SSL errors are just one type of thing you might need to debug.

Reply:
(The requesting party asked to add instructions on SSL debugging.)

Let's see, how about instead "Debug Presto to LDAP Server Issues"

Reply:
New commit pushed.

Reply:
That sounds good. Let's remove/generify the TLS/SSL part.

I also dislike repeating the part about `logging.properties`. How about we make that a link

```
If you need to debug issues with Presto communicating with the LDAP server,
you can change the :ref:`log level <log-levels>` for the LDAP authenticator:
```
You'll need to add an anchor above the "Log Levels" section in `deployment.rst`:
```
.. _log_levels:

Log Levels
^^^^^^^^^^
```

Reply:
Coming up.

Reply:
Et voila.